sjb/config/test_cases/azure_build_node_image_rhel.yml

```diff
@@ -0,0 +1,78 @@
+      <a href='https://github.com/openshift/release/blob/master/cluster/bin/local.sh'><code>cluster/bin/local.sh</code></a>
+      script.
+    default_value: "openshift/origin-ansible:latest"
+actions:
+  - type: "forward_parameters"
+    parameters:
+      - JOB_NAME
+      - BUILD_NUMBER
+      - CLUSTER_PROFILE
+      - OPENSHIFT_ANSIBLE_IMAGE
+
+  # calculate the instance prefix
+  - type: "host_script"
+    title: "determine the instance prefix"
+    script: |-
+      # date is enough for building image
+      export INSTANCE_PREFIX=$(date +%Y%m%d%H%M%S)
+      # write the standard variables for reuse
+      echo "INSTANCE_PREFIX=${INSTANCE_PREFIX:-}" >> INSTANCE_PREFIX
+      # hack around forwarding this to the other machine
+      cat INSTANCE_PREFIX | ssh -F ./.config/origin-ci-tool/inventory/.ssh_config openshiftdevel "cat >> /etc/environment"
+
+  - type: "script"
+    title: "clone repos"
+    script: |-
+      git clone --branch azure_provisioning https://github.com/kwoodson/openshift-ansible.git
+      git clone --branch azure https://github.com/pschiffe/release.git
+
+  - type: "script"
+    title: "build openshift installer image"
+    script: |-
+      pushd openshift-ansible
+      docker build -f images/installer/Dockerfile -t openshift/origin-ansible:latest .
+      popd
+
+  - type: "host_script"
+    title: "move secrets to remote host"
+    script: |-
+      rsync --copy-links --omit-dir-times --archive --rsh "ssh -F ./.config/origin-ci-tool/inventory/.ssh_config" /var/lib/jenkins/.azure/* openshiftdevel:/home/origin/release/cluster/test-deploy/${CLUSTER_PROFILE}/
+      ssh -F ./.config/origin-ci-tool/inventory/.ssh_config openshiftdevel 'sudo chown -R origin:origin-git /home/origin/release/cluster/test-deploy/${CLUSTER_PROFILE}/'
+      ssh -F ./.config/origin-ci-tool/inventory/.ssh_config openshiftdevel 'sudo chmod -R ug+rwX /home/origin/release/cluster/test-deploy/${CLUSTER_PROFILE}/'
+
+  - type: "script"
+    title: "build node image"
+    script: |-
+      pushd release/cluster/test-deploy/${CLUSTER_PROFILE}
+      TYPE=$CLUSTER_PROFILE ../../bin/ansible.sh ansible-playbook \
+        -e "openshift_test_repo=https://storage.googleapis.com/origin-ci-test/releases/openshift/origin/release-3.9/origin.repo" \
+        -e "debug_level=5" \
+        -e "openshift_azure_base_image_ns=magentaciimages" \
+        -e "openshift_azure_image=openshift-gi-${INSTANCE_PREFIX}" \
+        -e "openshift_azure_clusterid=magentaciimages" \
+        playbooks/${CLUSTER_PROFILE}/openshift-cluster/build_image.yml
+      popd
+
+  - type: "script"
+    title: "clean up resources"
+    script: |-
+      pushd release/cluster/test-deploy/${CLUSTER_PROFILE}
+      TYPE=$CLUSTER_PROFILE ../../bin/ansible.sh ansible-playbook \
+        -e "openshift_azure_image=openshift-gi-${INSTANCE_PREFIX}" \
+        -e "openshift_azure_clusterid=magentaciimages" \
+        playbooks/${CLUSTER_PROFILE}/openshift-cluster/remove_instance.yml
```

Comment:
Is this cleaning up the image that is built? How are we going to GC old base images?

Reply:
It's cleaning only the instance and it's resources (network interface, etc). The image stays (that's the point of the job). But we don't clean older images here, or in any other way.

Reply:
ack